reference/opt/shaders/frag/pixel-interlock-unordered.frag

```diff
@@ -1,6 +1,17 @@
 #version 450
+
+#ifdef GL_ARB_fragment_shader_interlock
+#extension GL_ARB_fragment_shader_interlock : enable
+#elif defined(GL_INTEL_fragment_shader_ordering)
+#extension GL_INTEL_fragment_shader_ordering : enable
+#endif
+
 #extension GL_ARB_fragment_shader_interlock : require
```

Comment:
again, why is this emitted?

Reply:
you asked me to emit this at the top when "detects usage of interlock" @devshgraphicsprogramming 
Did I misunderstand something?
```
#ifdef GL_ARB_fragment_shader_interlock
#extension GL_ARB_fragment_shader_interlock : enable
#else defined(GL_INTEL_fragment_shader_ordering)
#extension GL_INTEL_fragment_shader_ordering : enable
#endif
```